xbmc/settings/lib/SettingConditions.h

```diff
@@ -29,7 +29,7 @@
 class CSettingsManager;
 class CSetting;
 
-typedef bool (*SettingConditionCheck)(const std::string &condition, const std::string &value, const CSetting *setting);
+typedef bool (*SettingConditionCheck)(const std::string &condition, const std::string &value, const CSetting *setting, void *data);
```

Comment:
think the `void* data` can use a little doxy, what's done with it. I happen to know it know cause I've read the other code above.